lib/polyfill/symbol.js

```diff
@@ -0,0 +1,39 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+goog.provide('shaka.polyfill.Symbol');
+
+goog.require('shaka.log');
+goog.require('shaka.polyfill');
+
+/**
+ * @summary A polyfill to provide Symbol.prototype.description in all browsers.
+ * See: https://caniuse.com/mdn-javascript_builtins_symbol_description
+ * @export
+ */
+shaka.polyfill.Symbol = class {
+  /**
+   * Install the polyfill if needed.
+   * @export
+   */
+  static install() {
+    shaka.log.debug('Symbol.install');
+
+    // eslint-disable-next-line no-restricted-syntax
+    const proto = Symbol.prototype;
+
+    if (!('description' in proto)) {
+      Object.defineProperty(proto, 'description', {
+        get: () => {
+          const m = /\((.*)\)/.exec(this.toString());
```

Comment:
Using `this` in an arrow function refers to the enclosing scope, here meaning the `this` of `install()`, which I believe is `shaka.polyfill.Symbol`.

I think you'll need to replace this arrow function with a function or method.

I have tested toString() and the regex on Xbox, and it should work otherwise.